Packs/Endace/TestPlaybooks/Endace-Test.yml

```diff
@@ -0,0 +1,757 @@
+      timeframe:
+        simple: "3600"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 400
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "3":
+    id: "3"
+    taskid: fd78b2be-2058-4c83-869e-49eb5f3427b3
+    type: condition
+    task:
+      id: fd78b2be-2058-4c83-869e-49eb5f3427b3
+      version: -1
+      name: Verify Outputs
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Task.JobID
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Task.Status
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Task.Error
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: e49eb9ae-607d-490e-81f6-7708817a9b6b
+    type: regular
+    task:
+      id: e49eb9ae-607d-490e-81f6-7708817a9b6b
+      version: -1
+      name: endace-get-search-status
+      script: '|||endace-get-search-status'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      jobid:
+        simple: ${Endace.Search.Task.JobID}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: fc38fd5f-0f68-480d-8ef6-584a3bcb6e3c
+    type: condition
+    task:
+      id: fc38fd5f-0f68-480d-8ef6-584a3bcb6e3c
+      version: -1
+      name: Verify Outputs
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "6"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Response.JobID
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Response.Status
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Response.JobProgress
+            iscontext: true
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Endace.Search.Response.DataSources

```

Comment:
the test playbook fails for me because this is empty

Reply:
@Itay4 : can you please elaborate on this ?

Reply:
`Endace.Search.Response.DataSources` is empty on runtime, so the test playbook fails

Reply:
@Itay4 : The workflow in the playbook is Search, Archive and Download and they happen one after another in this playbook and next step depends upon fields from previous steps. If search is successful and data is found on EndaceProbe, then "Endace.Search.Response.DataSources" - won't be empty. Technically it is a valid outcome if search for packets fails which is indicated by TotalBytes context field. Can you help me understand if the workflow /decision making in the playbook is correct? if I need to add some checks ? Exactly similar playbook exists in playbooks folder with all checks in place. Will that be here rather? Let me know. Thanks.

Reply:
I ran the test playbook with out instance, and it failed because `DataSources` was empty

so it seems that it doesnt always return, and leads to inconsistent results on different instances, so need to be removed

Reply:
I need help here. I will discuss this with @fvigo during our conf call tomorrow.